escavenge.py

```diff
@@ -10,6 +10,9 @@ def __init__(self):
 
         self.maker = {
             "log out": lambda: self.game_maker.logout(),
+            "make team": lambda: self.game_maker.make_team(input),
```

Comment:
commands like "log out" and "make team" will need to be "logout" and "maketeam" with the new changes